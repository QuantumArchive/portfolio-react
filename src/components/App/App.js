import React from 'react'
import { hot } from 'react-hot-loader'

export const App = () => (
  <div>
    Hello
  </div>
)

export default hot(module)(App)
