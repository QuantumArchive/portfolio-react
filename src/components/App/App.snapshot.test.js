import React from 'react'
import expect from 'expect'
import renderer from 'react-test-renderer'
import { App } from './App'

describe('src/components/App/App.snapshot', () => {
  it('should render a dom element', () => {
    const tree = renderer.create(<App />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
