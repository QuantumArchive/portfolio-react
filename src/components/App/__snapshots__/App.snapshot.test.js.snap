// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/App/App.snapshot should render a dom element 1`] = `
<div>
  Hello
</div>
`;
